src/components/TestComponent.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import connectComponent from '@Lib/connect-component';
+import checkIsReducing from '../actions/initialAction';
+
+
+export const TestComponent = (props) => {
+  const { initialReducer: { isReducing }, dispatch } = props;
+  return (
+    <>
+      <h3>If redux is working, the value should toggle between true and false</h3>
+      <h4>
+        Hey! I am
+        {' '}
+        {isReducing ? 'true' : 'false'}
+      </h4>
+      <button type="submit" onClick={dispatch}>
+        Show
+        {' '}
+        {isReducing ? 'false' : 'true'}
+      </button>
+    </>
+  );
+};
+
+TestComponent.propTypes = {
+  dispatch: propTypes.func.isRequired,
+  initialReducer: propTypes.objectOf(propTypes.bool).isRequired,
+};
+export default connectComponent(TestComponent, { dispatch: () => checkIsReducing() });
```

Comment:
Great job @encodedBicoding. I have a suggestion, however. The property name for the function shouldn't be `dispatch` as this is the name of a function that comes with Redux. I had a hard time understanding what was happening there because I thought it was dispatching right from there. This is not a deal-breaker as this is a test component though. Great job once again.